src/tips/en/take-in-device-screenshots.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Take screenshots of your product in a device using DevTools
```

Comment:
A general comment about this tip: it's really great, and useful. I wonder if it's a bit long though. So far, we've kept tips on the site really quite short, with just 1 or 2 images at most. This one has a 6 images which are going to make the tip page quite a bit longer than the other ones. 
I'd love it if we could make the tip a bit shorter, so that it's faster to consume, and also easier to tweet.

I think the first 2 images are perhaps not necessary. What do you think?